core/src/Cuda/Kokkos_Cuda_View.hpp

```diff
@@ -174,11 +174,16 @@ struct CudaLDGFetch {
     return *this;
   }
 
+  KOKKOS_INLINE_FUNCTION
+  explicit CudaLDGFetch(const ValueType* const arg_ptr) : m_ptr(arg_ptr) {}
+
+  // For interface consistency with CudaTextureFetch even though second
+  // argument is unused here
```

Comment:
It's because `CudaTextureFetch` has to track a `cudaTextureObject_t` from the CUDA runtime library to provide it's functions. Not needed for CudaLDGFetch I guess.

Reply:
Meaning we can get rid of it?

Reply:
Now that https://github.com/kokkos/kokkos/pull/5623 is merged I think so, I'll rebase and see what I can see